pkg/kubelet/kuberuntime/kuberuntime_container.go

```diff
@@ -391,13 +441,20 @@ func (m *kubeGenericRuntimeManager) getPodContainerStatuses(uid kubetypes.UID, n
 			cStatus.Message = status.GetMessage()
 			cStatus.ExitCode = int(status.GetExitCode())
 			cStatus.FinishedAt = time.Unix(0, status.GetFinishedAt())
-		}
 
-		tMessage := getTerminationMessage(status, cStatus, annotatedInfo.TerminationMessagePath)
-		// Use the termination message written by the application is not empty
-		if len(tMessage) != 0 {
-			cStatus.Message = tMessage
+			// TODO: how is OOMKill represented in CRI?
```

Comment:
CRI doesn't define OOMKilled state clearly, but dockershim expose it by setting `ContainerStatus.Reason` to `OOMKilled`.